documentation/docs/05-modules.md

```diff
@@ -52,12 +52,13 @@ Stores are _contextual_ — they are added to the [context](https://svelte.dev/t
 
 Because of that, the stores are not free-floating objects: they must be accessed during component initialisation, like anything else that would be accessed with `getContext`.
 
-- `getStores` is a convenience function around `getContext` that returns `{ navigating, page, session }`. This needs to be called at the top-level or synchronously during component or page initialisation.
+- `getStores` is a convenience function around `getContext` that returns `{ navigating, url, route, session }`. This needs to be called at the top-level or synchronously during component or page initialisation.
 
-The stores themselves attach to the correct context at the point of subscription, which means you can import and use them directly in components without boilerplate. However, it still needs to be called synchronously on component or page initialisation when `$`-prefix isn't used. Use `getStores` to safely `.subscribe` asynchronously instead.
+The stores themselves attach to the correct context at the point of subscription, which means you can import and use them directly in components without boilerplate. However, it still needs to be called synchronously on component or page initialisation when the `$`-prefix isn't used. Use `getStores` to safely `.subscribe` asynchronously instead.
 
 - `navigating` is a [readable store](https://svelte.dev/tutorial/readable-stores). When navigating starts, its value is `{ from, to }`, where `from` and `to` both mirror the `page` store value. When navigating finishes, its value reverts to `null`.
```

Comment:
This needs updating. From what I saw of the code, I think `from` and `to` are now both `URL` instances, correct? This would mean that you no longer have access to the `params`. Is that fine?

Reply:
I actually think it's better — IIUC `params` could be incorrect in `to`, because you can't compute `params` until you've definitively selected a route

Reply:
(this was updated)